app/templates/components/supplementary-file-browser.hbs

```diff
@@ -5,45 +5,69 @@
     height="700"}}
 {{/file-renderer}}
 
-<div class="row">
-    <div class="col-sm-6">
-        <p class="f-w-lg p-b-md">{{selectedFile.name}}</p>
+<div class="row no-gutters">
+    <div class="col-lg-4">
+        <p id="selectedFileName" class="f-w-lg p-b-md">{{selectedFile.name}}</p>
     </div>
-    <div class="col-sm-6 text-nowrap supplemental-downloads">
+    <div class="col-lg-2 hidden-lg">
+        <p class="p-b-md">{{t "content.version"}}: {{selectedFile.currentVersion}}</p>
+    </div>
+    <div class="clearfix visible-xs-block"></div>
+    <div class="col-lg-6 col-xs-12 text-nowrap supplemental-downloads">
+        {{#if (eq selectedFile.id preprint.primaryFile.id)}}
+            <div class="dropdown lock-span">
+                <button class="btn btn-default btn-sm dropdown-toggle" type="button" id="previousVersionsDropdown"
+                  data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+                    {{t 'components.supplementary-file-browser.download_previous_versions'}}
+                    <span class="caret"></span>
+                </button>
+                <ul id="previousVersionsList" class="dropdown-menu" aria-labelledby="previousVersionsDropdown">
```

Comment:
This needs to fetch all or paginate or something. Possibly just page size huge (maybe 100), doubtful anyone would have more versions than that.
<img width="282" alt="screen shot 2018-08-28 at 10 03 44 am" src="https://user-images.githubusercontent.com/1322421/44731280-da67c580-aab0-11e8-8cce-66e84677143d.png">